app/src/main/res/layout/activity_main.xml

```diff
@@ -8,10 +8,8 @@
 
     <androidx.appcompat.widget.Toolbar
         android:id="@+id/toolbar"
-        style="@style/AppTheme.Toolbar"
```

Comment:
This and the theme are not needed anymore, since now we are properly setting the toolbar style in the theme (https://github.com/theaiscope/GDD-app/pull/51/files#diff-794f6ecce66f62dd7cd4f873b4dce4ecR8)